build-system/tasks/presubmit-checks.js

```diff
@@ -566,6 +566,7 @@ var forbiddenTermsSrcInclusive = {
   // Functions
   '\\.changeHeight\\(': bannedTermsHelpString,
   '\\.changeSize\\(': bannedTermsHelpString,
+  '\\.attemptChangeHeight\\(0\\)': 'please consider using `attemptCollapse()`',
```

Comment:
That's good point. But it is possible that an iframe request leads runtime to  call `attemptChangeSize(newWidth=0, newHeight=0)`. It is perfectly valid without setting element display to none afterwards.
The thing I want avoid is us explicitly put 0 as the input. `dev().assert` is too strict, maybe I can switch to `dev().warn`. honestly i feel presubmit check should be sufficient.